triciaswift/tastevine-server
# Added a Favorite Method, View, Url and List Method

## To Test
1. Open postman
2. Make sure the debugger is running
3. GET request to http://localhost:8000/favorites
4. An array of objects should return with the recipe serialized, along with the author of the recipe and the categories
